### DISCUSSION TITLE
How to disable the scrollbar

### DISCUSSION THREAD

**Discussion by natxopedreira** [forntoh/LcdMenu#254] [Q&A] [answered]
Hello

How to disable the bar that appears to the right side of the menu? I would like only the up/down arrow if necessary, imo looks cleaner

**Comment by forntoh** [comment, marked as answer]
It was a bug #247, fixed in **v5.3.1**

https://lcdmenu.forntoh.dev/overview/rendering/character-display.html#hide-the-arrows-only

**Reply by natxopedreira** [reply]
Perfect, thank you so much

**Comment by B1ackt34** [comment]
I want to hide arrows but this solution makes my ESP32 crash.
Is this the correct updated solution ?
